File: Python/medias_centralidade_variabilidade.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-
# @date: Jan/21   @author: @gkuster

import numpy as np
from scipy import stats

jogadores = [40000, 18000, 12000, 250000, 30000, 140000, 300000, 40000, 800000]

# Média
media = np.mean(jogadores)
media

# Mediana
mediana = np.median(jogadores)
mediana

# Quartis
quartis = np.quantile(jogadores, [0, 0.25, 0.5, 0.75, 1])
quartis

# Desvio padrão
desvio1 = np.std(jogadores)
desvio1

desvio2 = np.std(jogadores, ddof = 1)
desvio2

# Resumo da variável
stats.describe(jogadores)

File: R/medias_centralidade_variabilidade.r
# Salários 
jogadores = c(40000, 18000, 12000, 250000, 30000, 140000, 300000, 40000, 800000)

# Media
mean(jogadores)

# Mediana
median(jogadores)

# Quartis
quartis = quantile(jogadores)
quartis

# Por posição
quartis[0]
quartis[1]
quartis[2]
quartis[3]
quartis[4]

# Desvio padrão - Standart Desviation
sd(jogadores)

# Resumo da variável
summary(jogadores)